mne_bids/write.py

```diff
@@ -187,12 +190,60 @@ def _electrodes_tsv(raw, fname, kind, overwrite=False, verbose=True):
                             ('z', z),
                             ('size', sizes),
                             ])
-    else:
+    elif kind == 'eeg':
+        picks = pick_types(raw.info, eeg=True)
+        ch_locs = []
+        for ind, (name, _x, _y, _z) in enumerate(zip(names, x, y, z)):
+            if ind not in picks:
+                ch_locs.append([np.nan, np.nan, np.nan])
+            elif "n/a" in [_x, _y, _z]:
+                warn("{} electrode has n/a coordinate. "
+                     "MNE-bids is defaulting to np.nan. "
+                     "Please check electrodes.tsv.")
+                ch_locs.append([np.nan, np.nan, np.nan])
+            else:
+                ch_locs.append([_x, _y, _z])
+        ch_pos = dict(zip(names, np.array(ch_locs)))
+
+        # XXX: to be improved,
+        # see https://github.com/mne-tools/mne-bids/issues/264
+        # transform montage to head coordinate frame
+        montage = make_dig_montage(ch_pos=ch_pos,
+                                   coord_frame='unknown')
+        montage = transform_to_head(montage)
+
+        # get the coordinates again from transformed head
+        montage_chs = montage.ch_names
+        montage_dig = montage.dig
+        x, y, z = list(), list(), list()
+        for name in names:
+            if name in montage_chs:
+                ind = montage_chs.index(name)
+                digpoint = montage_dig[ind]
+                x.append(digpoint['r'][0])
+                y.append(digpoint['r'][1])
+                z.append(digpoint['r'][2])
+            else:
+                x.append('n/a')
+                y.append('n/a')
+                z.append('n/a')
         data = OrderedDict([('name', names),
                             ('x', x),
                             ('y', y),
                             ('z', z),
                             ])
+
+        # log messages to warn user of pitfalls of this writing
+        logger.info("EEG coords are converted to "
+                    "CapTrak system (i.e., head).")
```

Comment:
I recommend this string: `Using anatomical landmarks (NAS, LPA, RPA) to transform EEG electrode coordinates to CapTrak coordinate system.`